.github/dependabot.yml

```
@@ -1,11 +1,22 @@
 version: 2
 updates:
-- package-ecosystem: gomod
-  directory: "/"
-  schedule:
-    interval: weekly
-    time: "04:00"
-  open-pull-requests-limit: 10
-  allow:
-  - dependency-type: direct
-  - dependency-type: indirect
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: weekly
+      time: "04:00"
+    open-pull-requests-limit: 10
```

```suggestion
    open-pull-requests-limit: 10
    groups:
      go-modules:
        patterns:
          - "*"
```
Or with groups you could also consider dropping the `open-pull-requests-limit` as it will be rarely an isse